include/exadg/solvers_and_preconditioners/multigrid/coarse_grid_solvers.h

```diff
@@ -110,12 +110,14 @@ class MGCoarseKrylov
     {
       preconditioner.reset(new BlockJacobiPreconditioner<Operator>(coarse_matrix));
     }
-    else if(additional_data.preconditioner == MultigridCoarseGridPreconditioner::TrilinosAMG)
+    else if(additional_data.preconditioner == MultigridCoarseGridPreconditioner::AMG &&
+            additional_data.amg_data.amg_type == AMGType::Trilinos)
```

Comment:
An inner if-statement for Trilinos|Boomer would be better in my opinion and easier to read.

Reply:
I agree. done in commit [461275c](https://github.com/exadg/exadg/pull/58/commits/461275c128cb4e350d6591c1127901e1bf1b4a1c)